src/WCSimDetectorConfigs.cc

```diff
@@ -515,18 +675,18 @@ void WCSimDetectorConstruction::SetNuPrismShort_mPMTGeometry()
 	vessel_cyl_height = 38.*CLHEP::mm;    //option A, option B would be 277 mm
 	vessel_radius_curv = 342.*CLHEP::mm;  //needs to include the vessel thickness, as we construct from outside inwards.
 	vessel_radius = 254.*CLHEP::mm;
-	dist_pmt_vessel = 8*CLHEP::mm;
+	dist_pmt_vessel = 2*CLHEP::mm;      
```

Comment:
What's this change?

Reply:
Need to double check - might be a mistake in copying code or something. L628 change doesn't make sense to me

Reply:
This also seems to be reverting to an old value. I'm not sure why Lauren's version would be using the old values but I guess that's why it's now trying to change them back in this PR.